sample/python-env-secret/README.md

```diff
@@ -1,12 +1,13 @@
 # Deploying a Job
 This Job makes use of secret vars, it should be filled out before attempting to deploy.
 1. Copy env file from a template:
-```
-cp .env.dist .env
-```
-1. Edit `.env` file and fill out your secrets.
-1. Ensure the secret file is ignored by git.
-1. Deploy the sample.
+    ```
+    cp build.env.template build.env
+    cp runtime.env.template runtime.env
+    ```
+2. Edit `build.env` and `runtime.env` files and fill out your secrets.
+3. Ensure the secret file is ignored by git.
```

Comment:
I believe here the terminology is a bit overgrown. In line 1 there's `env file`, which is the same as `secret file` in line 3. Also, for someone with k8s background, it's a bit misleading to read about env vars and secret vars, as they are differently handled there. But here it seems env vars (like some service url) co-exist with secret vars (like auth tokens) in the same file. Not sure what do to here, just sharing my doubts.

Reply:
Alright, I'll call it "secret files". You're right, they no longer have anything to do with env vars (except being kept in `.env` files format).